converter/internal/prometheusconvert/testdata/scrape.yaml

```diff
@@ -11,12 +11,16 @@ scrape_configs:
     scrape_timeout: 5s
     static_configs:
       - targets: ["localhost:9090"]
```

Comment:
Can you make this a list of two targets to make sure that both of them get the app=foo label?

```yaml
static_configs:
- targets: ["localhost:9090", "localhost:9091"] 
  labels:
    app: 'foo'
```